extensions/sql-database-projects/src/dialogs/insertSqlBindingQuickpick.ts

```diff
@@ -0,0 +1,70 @@
+import * as vscode from 'vscode';
+import * as constants from '../common/constants';
+
+export async function launchInsertSqlBindingQuickpick(uri: vscode.Uri | undefined): Promise<void> {
+	if (!uri) {
+		// this command only shows in the command palette when the active editor is a .cs file, so we can safely assume that's the scenario
+		// when this is called without a uri
+		uri = vscode.window.activeTextEditor!.document.uri;
+	}
+
+	// 1. select input or output binding
+	const inputOutputItems: string[] = [constants.input, constants.output];
+
+	const selectedBinding = (await vscode.window.showQuickPick(inputOutputItems, {
+		canPickMany: false,
+		title: constants.selectBindingType,
+		ignoreFocusOut: true
+	}));
+
+	if (!selectedBinding) {
+		return;
+	}
+
+	// get all the azure functions in the file
+	// TODO: get actual functions. Need to add in sqltoolsservice first
+	const azureFunctions = ['af1', 'af2']; //await getAzureFunctions(uri);
```

Comment:
Quick Q : do we need to update db Proj for Aug? if so - we should not show the placeholders - right?

Reply:
Ah it wont show for ADS - got it

Reply:
yup and we can hide the command that shows this quick pick in case this doesn't get done before sql database projects gets released in vscode

Reply:
Updated name:
![image](https://user-images.githubusercontent.com/31145923/128219628-dcdba1e6-0067-4068-a3d9-dff6d7ab3b1c.png)